bug_reports/Golang_github.com-vmware-labs-yaml-jsonpath.md

```diff
@@ -0,0 +1,7 @@
+Results do not match other implementations
+
+The following queries provide results that do not match those of other implementations of JSONPath
+(compare https://cburgmer.github.io/json-path-comparison/):
```

Comment:
@cburgmer Not sure what this file is telling me. Could it be that my implementation doesn't match for a selector of the empty string??

Reply:
If this file is empty apart from the boilerplate, then it looks like you implementation is fully aligned with the current consensus! That's great news!

Reply:
Thanks! It took a few days of tweaking to align with the consensus, so I'm pleased the consensus didn't change in the meantime.